PWGHF/TableProducer/derivedDataCreatorDplusToPiKPi.cxx

```diff
@@ -289,7 +289,15 @@ struct HfDerivedDataCreatorDplusToPiKPi {
         if constexpr (IsMl) {
           std::copy(candidate.mlProbDplusToPiKPi().begin(), candidate.mlProbDplusToPiKPi().end(), std::back_inserter(mlScoresDplusToPiKPi));
         }
-        fillTablesCandidate(candidate, 0, massDplusToPiKPi, ct, y, flagMcRec, origin, swapping, flagDecayChanRec, mlScoresDplusToPiKPi);
+        auto trackprong0 = candidate.template prong0_as<TracksWPid>();
+        int const sign = trackprong0.sign();
+        uint8_t candFlag = 0;
+        if (sign > 0) {
+          candFlag |= BIT(0); // D+
+        } else if (sign < 0) {
+          candFlag |= BIT(1); // D-
+        }
```

Comment:
I assume that the sign cannot be 0 so there are only two options which are mutually exclusive so the second `if` is redundant, and so is the conversion to the bitmap.